qiita_db/test/test_user.py

```diff
@@ -75,7 +75,8 @@ def setUp(self):
             'receive_processing_job_emails': True,
```

Comment:
The issue with this test and my suggestion about using now() in the database is that there is going to be a small difference on the tests so they will need to be adjusted.